symbolic/src/Data/Macaw/Symbolic.hs

```diff
@@ -827,7 +827,7 @@ mkFunRegCFG archFns halloc nm posFn fn = crucGenArchConstraints archFns $ do
     ng <- mmNonceGenerator
     inputAtom <- mmExecST $ CR.mkInputAtoms ng entryPos (Empty :> regType) >>= \case
       Empty :> atm -> return atm
-      _ -> error "Error creating input atom for mkFunRegCFG"
+      _ -> error "Error creating input atom for mkFunRegCFG"  -- TODO: fix warning
```

Comment:
```
src/Data/Macaw/Symbolic.hs:830:7: error: [-Woverlapping-patterns, -Werror=overlapping-patterns]
    Pattern match is redundant
    In a case alternative: _ -> ...
    |
830 |       _ -> error "Error creating input atom for mkFunRegCFG"
    |   
```

Reply:
Also resolved by https://github.com/GaloisInc/macaw/pull/172

Reply:
This has the same problem as the line above.